openmetadata-ui/src/main/resources/ui/src/utils/TagsUtils.ts

```diff
@@ -151,3 +151,9 @@ export const fetchTagsAndGlossaryTerms = async () => {
 
   return tagsAndTerms;
 };
+
+export const isSystemTags = (tag: string) => {
+  const systemTag = 'tier';
```

Comment:
As discussed please improve systemTag check as it will also disable for the tags which will contain `tier` within it